Build failing on an import error:
```
Traceback (most recent call last):
  File "./manage.py", line 172, in <module>
    execute_from_command_line(sys.argv)
  File "/vendor/lib/python2.7/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/vendor/lib/python2.7/site-packages/django/core/management/__init__.py", line 338, in execute
    django.setup()
  File "./manage.py", line 56, in _setup_once
    _setup_once.setup(*args, **kw)
  File "/vendor/lib/python2.7/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/vendor/lib/python2.7/site-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models()
  File "/vendor/lib/python2.7/site-packages/django/apps/config.py", line 202, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/local/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/mnt/commcare-hq-ro/corehq/apps/commtrack/models.py", line 10, in <module>
    from corehq.form_processor.change_publishers import publish_ledger_v1_saved
  File "/mnt/commcare-hq-ro/corehq/form_processor/change_publishers.py", line 5, in <module>
    from corehq.apps.change_feed import data_sources, topics
  File "/mnt/commcare-hq-ro/corehq/apps/change_feed/data_sources.py", line 17, in <module>
    from corehq.util.datadog.util import (
ImportError: No module named util
```

--- a/corehq/apps/change_feed/data_sources.py
+++ b/corehq/apps/change_feed/data_sources.py
@@ -14,7 +14,7 @@ from corehq.form_processor.document_stores import (
 )
 from corehq.util.couch import get_db_by_doc_type
 from corehq.util.couchdb_management import couch_config
-from corehq.util.datadog.util import (
+from corehq.util.datadog.utils import (
     case_load_counter,
     form_load_counter,
     ledger_load_counter,

--- a/corehq/apps/change_feed/tests/test_data_sources.py
+++ b/corehq/apps/change_feed/tests/test_data_sources.py
@@ -10,8 +10,11 @@ from corehq.apps.change_feed.exceptions import UnknownDocumentStore
 from corehq.apps.locations.document_store import ReadonlyLocationDocumentStore
 from corehq.apps.sms.document_stores import ReadonlySMSDocumentStore
 from corehq.form_processor.document_stores import (
-    ReadonlyFormDocumentStore, ReadonlyCaseDocumentStore, ReadonlyLedgerV2DocumentStore,
-    LedgerV1DocumentStore
+    DocStoreLoadTracker,
+    LedgerV1DocumentStore,
+    ReadonlyCaseDocumentStore,
+    ReadonlyFormDocumentStore,
+    ReadonlyLedgerV2DocumentStore,
 )
 from corehq.util.exceptions import DatabaseNotFound
 from corehq.util.test_utils import generate_cases
@@ -52,4 +55,6 @@ class DocumentStoreTests(SimpleTestCase):
 def test_get_document_store(self, source_type, source_name, expected, sql_domain=False):
     with override_settings(TESTS_SHOULD_USE_SQL_BACKEND=sql_domain):
         store = get_document_store(source_type, source_name, 'domain')
+    if isinstance(store, DocStoreLoadTracker):
+        store = store.store
     self.assertEqual(store.__class__, expected)

